app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/CameraUtils.kt

```diff
@@ -0,0 +1,209 @@
+package com.example.android.architecture.blueprints.todoapp.data
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.*
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import com.example.android.architecture.blueprints.todoapp.R
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.io.InputStream
+import java.util.*
+
+fun getCircleBitmap(color: Int, size: Int, activityContext: Context): Drawable {
+    val a = Bitmap.createBitmap(size, size, Bitmap.Config.ARGB_8888)
+    val canvas = Canvas(a)
+    val paint = Paint()
+    paint.isAntiAlias = true
+    paint.color = color
+    paint.style = Paint.Style.FILL
+    canvas.drawBitmap(a, 0f, 0f, paint)
+    canvas.drawCircle(size / 2f, size / 2f, size / 2f, paint)
+    return BitmapDrawable(activityContext.resources, a)
+}
+
```

Comment:
If you are using context object, give him the name context, but not the activity. This may lead misunderstanding